node/sc/api_bridge.go

```diff
@@ -179,8 +202,22 @@ func (sb *SubBridgeAPI) SubscribeBridge(cBridgeAddr, pBridgeAddr common.Address)
 	return nil
 }
 
-// UnsubscribeBridge disables the event subscription of the given child/parent chain bridges.
-func (sb *SubBridgeAPI) UnsubscribeBridge(cBridgeAddr, pBridgeAddr common.Address) error {
+func (sb *SubBridgeAPI) SubscribeBridge(cBridgeAddrOrAlias, pBridgeAddrOrEmpty *string) error {
+	cBridgeAddrOrAliasStr, pBridgeAddrOrEmptyStr := stringDeref(cBridgeAddrOrAlias), stringDeref(pBridgeAddrOrEmpty)
+
+	var err error
+	bridgeAlias, cBridgeAddr, pBridgeAddr, _ := parseBridgeAddrWithAlias(cBridgeAddrOrAliasStr, pBridgeAddrOrEmptyStr)
+	if bridgeAlias != "" {
+		cBridgeAddr, pBridgeAddr, err = sb.subBridge.bridgeManager.getAddrByAlias(bridgeAlias)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
This logic is repeated when `parseBridgeAddrWithAlias` is called. 
How about processing `getAddrByAlias` inside of `parseBridgeAddrWithAlias` ?
Then, `parseBridgeAddrWithAlias` is gonna return `(cBridgeAddr, pBridgeAddr, args, err)`

Reply:
Good suggestion. Refactored.